2024-09-30 22:23:04,261 - root - INFO - Restored all variables from .//models/model.pt
2024-09-30 22:23:04,263 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-30 22:23:22,923 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 468.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 343.69 MiB is free. Process 137803 has 23.35 GiB memory in use. Of the allocated memory 22.42 GiB is allocated by PyTorch, and 627.66 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-30 22:23:22,940 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 468.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 343.69 MiB is free. Process 137803 has 23.35 GiB memory in use. Of the allocated memory 22.42 GiB is allocated by PyTorch, and 627.66 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-30 22:24:30,346 - root - INFO - Restored all variables from .//models/model.pt
2024-09-30 22:24:30,347 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-30 22:24:48,392 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 468.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 343.69 MiB is free. Process 139960 has 23.35 GiB memory in use. Of the allocated memory 22.42 GiB is allocated by PyTorch, and 627.66 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-30 22:24:48,408 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 468.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 343.69 MiB is free. Process 139960 has 23.35 GiB memory in use. Of the allocated memory 22.42 GiB is allocated by PyTorch, and 627.66 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-30 22:26:47,661 - root - INFO - Restored all variables from .//models/model.pt
2024-09-30 22:26:47,662 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-30 22:27:03,742 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 468.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 343.69 MiB is free. Process 143137 has 23.35 GiB memory in use. Of the allocated memory 22.42 GiB is allocated by PyTorch, and 627.66 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-30 22:27:03,757 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 468.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 343.69 MiB is free. Process 143137 has 23.35 GiB memory in use. Of the allocated memory 22.42 GiB is allocated by PyTorch, and 627.66 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-30 22:27:57,804 - root - INFO - Restored all variables from .//models/model.pt
2024-09-30 22:27:57,805 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-30 22:30:00,072 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: 
